manifests/init.pp

```diff
@@ -141,7 +141,7 @@
   }
 
   File {
-    backup  => false,
+    backup  => $backup,
```

Comment:
I don't think this is needed to work with the static compiler. None of these files have sources pointing at the puppet fileserver.